src/H5PEditor.ts

```diff
@@ -60,12 +62,12 @@ export default class H5PEditor {
         public config: IH5PConfig,
         public libraryStorage: ILibraryStorage,
         public contentStorage: IContentStorage,
-        public temporaryStorage: ITemporaryFileStorage
+        public temporaryStorage: ITemporaryFileStorage,
+        private urlGenerator: IUrlGenerator = new UrlGenerator(config)
```

Comment:
Could you please add this new parameter to docs/h5p-editor-constructor.md? I think it would be great if you explained what a typical use case for using a custom URL generator is.